src/components/data/SavedColumnSettings.js

```diff
@@ -127,17 +127,19 @@ const useSavedColumnSettings = ({ workspaceId, snapshotName, entityMetadata, ent
       encodeColumnSettings(columnSettings),
       allColumnSettings
     )
-    await updateSavedColumnSettings(newColumnSettings)
+    await updateAllSavedColumnSettings(newColumnSettings)
   }
 
   const deleteSavedColumnSettings = async columnSettingsName => {
     // Re-fetch column settings to reduce the chances of overwriting changes made by other users since we first loaded settings.
     const allColumnSettings = await getAllSavedColumnSettings()
     const newColumnSettings = _.omit(`${entityTypeKey}.${columnSettingsName}`, allColumnSettings)
-    await updateSavedColumnSettings(newColumnSettings)
+    await updateAllSavedColumnSettings(newColumnSettings)
   }
 
   return {
+    getAllSavedColumnSettings,
+    updateAllSavedColumnSettings,
```

Comment:
Expose functions to get/set saved column settings for all tables in the workspace. Previously, for convenience, `useSavedColumnSettings` only exposed functions to get/set saved column settings for the table specified by the `entityType` parameter.